template/src/App.vue

```diff
@@ -4,7 +4,7 @@
     {{#router}}
     <router-view/>
     {{else}}
-    <hello-world/>
+    <HelloWorld/>
```

Comment:
Instead of changing this why not tell people that Vue automatically converts 

HTML is preferred kebab-case by developers/designers worldwide!